20240308 faq temporary fix (#961)
utelecon/utelecon.github.io: src/pages/en/faq/index.md

## Questions regarding classes in general

### Q. How do I take attendance?
Some options for taking attendance are:

> * Use the UTOL attendance management feature (the explanation page is under preparation)

Reviewer: ページ自体は生えていて，準備中ですと書かれています．
```suggestion
* Use the UTOL [attendance management](/en/utol/lecturers/attendances/) feature
```

* If you can sign into UTokyo Account User Menu and UTOL but not UTAS:
  * Your UTAS account may not have been issued. Please send your 10-digit common ID to the faculty office to check if your UTAS account has been issued.
* If you cannot sign into these systems:
  * Reset your UTokyo Account <a href="https://utacm.adm.u-tokyo.ac.jp/webmtn/multi/jpn/reset.html" target="_blank"> password</a>.
* Reference: [UTAS](/utas)

### Q. 「[Error] あなたは現在このシステムを利用することができません」is shown and sign-in is unavailable. What can I do?
* This may occur during affiliation transition, for example, from undergraduate to graduate. Please try again later.
* If this does not apply to your situation, please contact the faculty or graduate school you belong to.
* UTAS accounts of some faculty members (especially those who are not teaching staff) are not registered. Contact your faculty for more information.

## UTOL
### Q. What is UTOL?
* UTOL is the learning management system of UTokyo.
* Features of UTOL include lecture materials downloading, assignment submission, online examinations, surveys, and forum discussion for courses registered.

> * Reference: [UTOL](/utol)

Reviewer: ```suggestion
* Reference: [UTOL](/utol/)
```

    1. Open a Google App such as Gmail, and press the icon in the upper right corner (either your own icon, or a human symbol if you have not set one up).
    2. Tap "Add another account" to add your Google account.
    3. You will be prompted to enter your email address and password, so enter your ECCS Cloud Email email address and password that you set up above.

### Q. I forgot my password / I cannot change my password. What can I do?

* See [ECCS Cloud Email – “Initial Setup Procedures”](/en/google/#initial-setup-procedures), and follow the instructions in “Task: Set your password and email address”.

### Q. What is the difference among `10-digits-ID@utac`, `10-digits-ID@g.ecc`, and `xxxx@g.ecc` (email address named after your choice) email addresses?
* `10-digits-ID@utac.u-tokyo.ac.jp`(e.g. `1234567890@utac.u-tokyo.ac.jp`)
  * Although it resembles an email address, it cannot be used for sending emails. However, the system may still send notifications or other messages to this address. For instance, regarding Zoom or UTokyo Slack, system-generated emails sent to this address can be received in each individual’s mailbox of ECCS Cloud Email.

>   * You might use your UTokyo Account as your username when logging into the system. The 10-digit number preceding the `@` symbol is known as the common ID. Some system require you to enter only this common ID when logging in.

Reviewer 2: ```suggestion
  * This may be used as your user name when you sign in to the system with your UTokyo Account. The 10-digit number before @ is called the common ID, and some systems require only this common ID to sign in.
```

>   * You might use your UTokyo Account as your username when logging into the system. The UTokyo Account’s username, often referred to as the common ID, is a 10-digit number. This 10-digit number, to explicitly represent it as an ID from the University of Tokyo, can also be appended with @utac.u-tokyo.ac.jp.

Reviewer: 日本語のコメント参照
// ところで1文目は文意が取れないです
